docs/book/src/usage/api.rst

```diff
@@ -27,6 +27,96 @@ By default it will bind the service on **localhost:8090**. If you want to change
 
     $ ./utils/api.py --host 0.0.0.0 --port 1337
 
+Web deployment
+--------------
+
+While the default method of starting the API server works fine for many cases, 
+some users may wish to deploy the server in a robust manner. This can be done 
+by exposing the API as a WSGI application through a web server. This section shows 
+a simple example of deploying the API via `uWSGI`_ and `Nginx`_. These 
+instructions are written with Ubuntu GNU/Linux in mind, but may be adapted for 
+other platforms.
+
+This solution requires uWSGI, the uWSGI Python plugin, and Nginx. All are available as packages::
+
+    $ sudo apt-get install uwsgi uwsgi-plugin-python nginx
+
+uWSGI setup
+^^^^^^^^^^^
+First, use uWSGI to run the API server as an application.
+
+To begin, create a uWSGI configuration file at ``/etc/uwsgi/apps-available/cuckoo-api.ini``::
+
+    [uwsgi]
+    plugins = python
+    chdir = /home/cuckoo/cuckoo
+    file = utils/web.py
+    uid = cuckoo
+    gid = cuckoo
+
+This configuration inherits a number of settings from the distribution's 
+default uWSGI configuration, loading ``web.py`` from the Cuckoo installation 
+directory. If Cuckoo is installed in a different path, adjust the configuration 
+(the *chdir* setting, and perhaps the *uid* and *gid* settings) accordingly.
+
+Enable the app configuration and start the server::
+
+    $ sudo ln -s /etc/uwsgi/apps-available/cuckoo-api.ini /etc/uwsgi/apps-enabled/
+    $ sudo service uwsgi start cuckoo-api    # or reload, if already running
+
+.. note::
+
+   Logs for the application may be found in the standard directory for distribution
+   app instances, i.e.:
+
+   ``/var/log/uwsgi/app/cuckoo-api.log``
+
+   The UNIX socket is created in a conventional location as well:
+
+   ``/run/uwsgi/app/cuckoo-api/socket``
+
+Nginx setup
+^^^^^^^^^^^
+
+With the API server running in uWSGI, Nginx can now be set up to run as a web
+server/reverse proxy, backending HTTP requests to it.
+
+To begin, create a Nginx configuration file at ``/etc/uwsgi/apps-available/cuckoo-api.ini``::
```

Comment:
This should be ``/etc/nginx/sites-available/cuckoo-api`` as used later as well :)